apps/web-portal-client/lib/components/layout/sidebar/SidebarLayout.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```

Comment:
Looks like `over engineering` to me, just move all that code to `SideBar` and remove this one